sympy/solvers/solvers.py

```diff
@@ -2613,8 +2541,8 @@ def _tsolve(eq, sym, **flags):
     >>> from sympy.solvers.solvers import _tsolve as tsolve
     >>> from sympy.abc import x
 
-    >>> tsolve(3**(2*x + 5) - 4, x)
-    [-5/2 + log(2)/log(3), (-5*log(3)/2 + log(2) + I*pi)/log(3)]
+    >>> set(tsolve(3**(2*x + 5) - 4, x))
+    {(-5*log(3)/2 + log(2) + I*pi)/log(3), -5/2 + log(2)/log(3)}
```

Comment:
Is this doctest reliable?

I would have expected the set to print in different orders on different runs.

Reply:
The test seems to work okay when I run it several times. Maybe that's because the SymPy printer makes the order deterministic...

Reply:
It is written to be reliable since `{1,2}=={2,1}`

Reply:
Yes, but with standard Python repr printing the order when printed is non-deterministic and doctest matches the text output directly (rather than doing any set comparison).